src/gluonts/model/tft/_estimator.py

```diff
@@ -155,161 +155,106 @@ def __init__(
         )
 
     def create_transformation(self) -> Transformation:
-        transforms = (
-            [AsNumpyArray(field=FieldName.TARGET, expected_ndim=1)]
-            + (
-                [
-                    AsNumpyArray(field=name, expected_ndim=1)
-                    for name in self.static_cardinalities.keys()
-                ]
-            )
-            + [
-                AsNumpyArray(field=name, expected_ndim=1)
-                for name in chain(
-                    self.static_feature_dims.keys(),
-                    self.dynamic_cardinalities.keys(),
-                )
-            ]
-            + [
-                AsNumpyArray(field=name, expected_ndim=2)
-                for name in self.dynamic_feature_dims.keys()
-            ]
-            + [
-                AddObservedValuesIndicator(
-                    target_field=FieldName.TARGET,
-                    output_field=FieldName.OBSERVED_VALUES,
-                ),
-                AddTimeFeatures(
-                    start_field=FieldName.START,
-                    target_field=FieldName.TARGET,
-                    output_field=FieldName.FEAT_TIME,
-                    time_features=self.time_features,
-                    pred_length=self.prediction_length,
-                ),
-            ]
+        transform = AsNumpyArray(field=FieldName.TARGET, expected_ndim=1)
+
+        for name in self.static_cardinalities.keys():
+            transform += AsNumpyArray(field=name, expected_ndim=1)
+
+        for name in chain(
+            self.static_feature_dims.keys(),
+            self.dynamic_cardinalities.keys(),
+        ):
+            transform += AsNumpyArray(field=name, expected_ndim=1)
+
+        for name in self.dynamic_feature_dims.keys():
+            transform += AsNumpyArray(field=name, expected_ndim=2)
+
+        transform += AddObservedValuesIndicator(
+            target_field=FieldName.TARGET,
+            output_field=FieldName.OBSERVED_VALUES,
```

Comment:
Could we use defaults here? Especially for this transform, it seems like `target_field` and `output_field` almost never change.

Reply:
Yes, I forgot to update this: `AddObservedTargetIndicator` fixes those arguments